{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -29,13 +29,13 @@ dev = [
 ]
 doc = [
     "sphinx>=4",
-    "furo",
-    "myst-parser",
+    "sphinx-book-theme>=0.3.3",
+    "myst-nb",
```

Comment:
The developer docs mention nbsphinx. Can you please update that section?

Reply:
Done.